### DISCUSSION TITLE
way to show voxel from the top view layer by layer

### DISCUSSION THREAD

**Discussion by Behold3D** [guillaumechereau/goxel#313] [Ideas]
I see an application where a user can:

1.  import an obj, 
2. have it voxelated according to the inputted voxel size 
3.  modify the material and color
4. and view the model by voxel height (layer by layer)

Is that currently possible?

**Comment by guillaumechereau** [comment]
The import should be possible (although not very polished).

Currently there is no simple way to view layer by layer, but I think one nice way to support that would be to allow layer operations (like substract), so we could add a top layer filled with a substracting block and make it move one voxel at a time.

**Comment by Behold3D** [comment]
That would be great. Is it difficult to do?

**Comment by guillaumechereau** [comment]
I am not sure, I never tried.  But in theory no, since the internal voxel volume code already supports subtraction.

**Comment by Behold3D** [comment]
Is there anything I could do to help?

________________________________

**Comment by guillaumechereau** [comment]
if you know how to code you could have a try at it.  Otherwise you can make
some fake screenshots of how the ui could look like, if you have some
design skills.


On Fri, Jul 21, 2023 at 6:31 PM Behold3D ***@***.***> wrote:

> Is there anything I could do to help?
>
> ________________________________
> From: Guillaume Chereau ***@***.***>
> Sent: Thursday, July 20, 2023 7:27 PM
> To: guillaumechereau/goxel ***@***.***>
> Cc: Behold3D ***@***.***>; Author ***@***.***>
> Subject: Re: [guillaumechereau/goxel] way to show voxel from the top view
> layer by layer (Discussion #313)
>
>
> I am not sure, I never tried. But in theory no, since the internal voxel
> volume code already supports subtraction.
>
> —
> Reply to this email directly, view it on GitHub<
> https://github.com/guillaumechereau/goxel/discussions/313#discussioncomment-6504173>,
> or unsubscribe<
> https://github.com/notifications/unsubscribe-auth/ACM5CBGQITTJTY6HQZD4HW3XRG5GDANCNFSM6AAAAAAZRXNTQE>.
>
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/guillaumechereau/goxel/discussions/313#discussioncomment-6508465>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAA2JH7OX3354I23MO72U7TXRJLBZANCNFSM6AAAAAAZRXNTQE>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>
-- 
Sent from my iPad.

**Comment by Behold3D** [comment]
Design skills I have!

Lino

**Comment by Behold3D** [comment]
i haven't forgotten. (well ok.. i sorta did) I am tied up with a project. I will circle back soon. Thanks for your patience.
